systemd/rqbit.conf

```diff
@@ -0,0 +1,36 @@
+#RQBIT_LOG_LEVEL_CONSOLE=
```

Comment:
This is just a listing of all supported environment variables. It doesn't need to exist, it's just a nice way to let the user know what they can configure (I'll need to add documentation/defaults, this is just a first pass).